README.md

```diff
@@ -6,8 +6,34 @@ This is the security scan using ZAP proxy.
 
 ### Prerequisites
 
+The following software needs to be installed:
+
 * [Docker](https://www.docker.com)
 
+### Local environment setup
+
+In addition to above, a link to integration tests should exist as security scan runs integration tests through ZAP proxy. Link can be created using the following command:
+
+```bash
```

Comment:
this isn't required on jenkins though right?

Reply:
It isn't because Jenkins does a checkout to that directory so it has a local copy.